src/styles/header.scss

```diff
@@ -0,0 +1,71 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
```

Comment:
It's a good practice to use semantic HTML elements. If the `.logo` class is for the site's logo, consider using an `<a>` tag if it's a link, or `<img>` if it's just an image, rather than a generic `<div>` or `<span>`.